api/tests/src/data_migration/transformation/subtask/test_transform_funding_category.py

```diff
@@ -61,7 +61,7 @@ def test_process_funding_categories(self, db_session, transform_funding_category
         )
 
         opportunity_summary_forecast_hist = f.OpportunitySummaryFactory.create(
```

Comment:
If we were setting the revision number before, we should just delete these records since they're tests for historical scenarios.